lib/circleci_solidus_extensions.rb

```diff
@@ -0,0 +1,193 @@
+# frozen_string_literal: true
+
+require 'circleci'
+require 'erb'
+
+module SolidusExtensions
+  module CircleCi
+    class Project
+      include ERB::Util
+      attr_reader :org, :repo
+
+      def initialize(org, repo, branches = nil)
+        @org = org
+        @repo = repo
+        @branches = branches || ['master']
+      end
+
+      def github_url
+        "https://github.com/#{org}/#{repo}"
+      end
+
+      def branches
+        @branches.map do |name|
+          Branch.new(self, name)
+        end
+      end
+
+      def circleci_project
+        ::CircleCi::Project.new(org, repo)
```

Comment:
If I understand correctly, this must be prefixed with `::` in order to avoid wrong constant resolution. I wonder if it may make sense changing this namespace `SolidusExtensions::CircleCi::Project` to something else, so there's no naming clash.

Reply:
Yes, we could but I don't know if it's a good idea to use a different name just to avoid clashing. Also, I don't think it should be considered a bad thing being explicit on the class to call here.